aip/general/0185.md

```diff
@@ -24,7 +24,7 @@ Note: The use of the term "major version number" above is taken from
 [semantic versioning][]. However, unlike in traditional semantic versioning,
 Google APIs **must not** expose minor or patch version numbers. For
 example, Google APIs use `v1`, not `v1.0`, `v1.1`, or `v1.4.2`. From a user's
-perspective, major versions are updated in place, and users receive new
+perspective, minor/patch versions are updated in place, and users receive new
```

Comment:
I understand what you are going for here, but I think the description is different with this change.

The intention is to convey that a major version is updated in place, rather than getting a new, minor/patch bumped version for compatible changes. 

Perhaps something along the lines of...
```suggestion
perspective, major versions are updated in place with minor/patch equivalent changes, and users receive new
```

(correct for line wrapping of course)